docs/how-to/required-software.md

```diff
@@ -0,0 +1,96 @@
+# How To - Required software
+
+## Introduction
+
+This guide shows you how to setup a Mac for iOS development.
+
+You will need a Mac in order to run this. It will not work on a Windows or Linux device.
+
+## Xcode
+
+You'll need an Apple ID in order to download Xcode. Download it using **[this link](https://developer.apple.com/download/applications)**. You will need to become an Apple Developer (you will be prompted to do this, and agree to their terms and conditions) - this is free.
+
+Select the latest stable version of Xcode (i.e Xcode 15.0.1, not the Beta version), then click `Download Xcode XX.X.X (2.96GB)` to download a compressed file:
```

Comment:
Why do we specify 15.0.1 and then just say XX.X.X here? Also, when we say XX.X.X, is there any benefit in saying the exact file size as well (i.e. 2.96GB) if the file size could potentially change with the version?

Reply:
The specified version, and later the file size, are more examples that can be more easily matched up with what should be clicked in the image below. There's no reason to include them otherwise if you feel it's already clear as like you said, these numbers will likely change

Reply:
I just think it reads strangely to see an exact version used, then to use the vague XX.X.X, together with an _exact_ file size. Would it be sufficient to just say
"Select the latest stable version of Xcode (i.e not a beta release), then click `Download Xcode X.X.X (X.XGB)` to download a compressed file:"
This way we wouldn't have to worry about this doc becoming outdated as newer versions are released?

Reply:
And as you said, the screenshot provides an _example_ of what this screen will look like if anything is still unclear to someone reading.